### PR TITLE
add placeholder OpenShiftConfig, enable acs-engine orchestrators output

### PR DESCRIPTION
**Pull request by jim-minter**
add placeholder OpenShiftConfig, enable acs-engine orchestrators output

### CODE REVIEW

File: pkg/api/vlabs/types.go [hunk 1]
```diff
@@ -276,6 +277,13 @@ type DcosConfig struct {
 	DcosWindowsBootstrapURL string `json:"dcosWindowsBootstrapURL,omitempty"`
 }
 
+// OpenShiftConfig contains the Kubernetes config structure, containing
+// Kubernetes specific configuration
+type OpenShiftConfig struct {
+	// TODO: fill this out, remove nested KubernetesConfig
+	KubernetesConfig
```
Comment (kargakis) [inline pkg/api/vlabs/types.go]: Hrm, I don't think the standard JSON library supports inlining ...

Comment (kargakis) [reply pkg/api/vlabs/types.go]: Or is the issue with `,inline` a different one?

Comment (jim-minter) [reply pkg/api/vlabs/types.go]: I think this works as I intended it to - the enclosed struct has no name so OpenShiftConfig serialises just like KubernetesConfig, with the option of adding additional members if needed.  https://play.golang.org/p/Gqei3FnZslT

**Comment by jim-minter** [comment]
closing for #3

**Pull request closed without merging**